README.md

```diff
@@ -289,7 +289,39 @@ VERBOSE=1 node app
 
 ## Setting `baseUrl` and naming environments
 
-By default, `sites` come with three url fields in their schema that correspond to three server environments: `dev`, `staging`, and `prod`. From the Dashboard, you are able to set the `baseUrl` property of each site within each environment. To let `apostrophe-multisite` know what environment it is currently running in, add the property `env` to your server's `data/local.js` file.
```

Comment:
Actually the usual thing is to set the ENV environment variable to dev, staging or prod. local.js does work but multisite tends to be deployed to the cloud and clouds tend to prefer environment based configuration. So I'd document the ENV approach.

Reply:
Ok, I tried to follow your suggestion. If you have a better formulation, let me know please